### PR TITLE
ref(brigade.js): updates per brigade-utils helpers

### PR DESCRIPTION
**Pull request by vdice**
* Refactors Brigade CI with the help up brigade-utils
* Enables check suite runs on `master` branch (handy to see status of branch at a glance; will enable build badge from Checks API down the line)

### CODE REVIEW

File: brigade.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ events.on("push", (e, p) => {
 events.on("check_suite:requested", runSuite);
 events.on("check_suite:rerequested", runSuite);
 events.on("check_run:rerequested", checkRequested);
-events.on("issue_comment:created", handleIssueComment);
-events.on("issue_comment:edited", handleIssueComment);
+events.on("issue_comment:created", (e, p) => Check.handleIssueComment(e, p, runSuite));
```
Comment (radu-matei) [inline brigade.js]: Importing the `Check` object is also needed:

```
const { Check } = require("@brigadecore/brigade-utils");
```

**Comment by radu-matei** [comment]
```
prestart: installing @brigadecore/brigade-utils@0.4.1
[brigade] brigade-worker version: 1.2.1
[brigade:k8s] Creating PVC named brigade-worker-01e3azf8dxj1zqa6vaesnp5z15
[brigade:app] ReferenceErrorCheck is not defined
```

File: brigade.js (hunk 1 above)
Comment (vdice) [reply brigade.js]: Whoops!  Added.  Thanks!

**Pull request merged**